dashboard/i18n/en.po

```diff
@@ -0,0 +1,16 @@
+msgid ""
+msgstr ""
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: POEditor.com\n"
+"Project-Id-Version: plgd #dashboard\n"
+"Language: en\n"
+
+#. [appName]
+#. defaultMessage is:
+#. Plgd UI
+#: ./i18n/template.json
+msgid "Plgd UI"
```

Comment:
plgd is written in lowercase. And, this is not plgd UI but plgd Dashboard

Reply:
Yeah, this is a template .po file, will be overwritten by our translation tool.